docs/hcc.md

```diff
@@ -0,0 +1,48 @@
+# git-source-control for Health Connect Cloud
+
+## Development Workflow
+
+git-source-control has two main use modes: Basic and Expert. For Health Connect Cloud, we recommend beginning with basic mode, since it includes all of the necessary functionality for proper source control development while being easy to use for those with little development / source control experience. The recommended workflow for HCC is to create featyre branches, merge them into development, and then git pull the changes in the shared development namespace. The rest of this section explains this workflow more in depth for those with less source control experience. 
+
+### Feature branches
```

Comment:
We came up with the term "Interface branch" in a different discussion you may or may not have been part of; let's use that.